test/unit/size_classes.c

```diff
@@ -79,6 +80,44 @@ TEST_BEGIN(test_size_classes) {
 }
 TEST_END
 
+TEST_BEGIN(test_grow_slow_size_classes) {
+	test_skip_if(!config_limit_usize_gap);
+	/*
+	 * Override runtime option for this test to test usize calculation when
+	 * sz_limit_usize_gap_enabled() is true.
+	 */
+	opt_limit_usize_gap = true;
```

Comment:
This is quite risky. Changing the setting in the middle of the run means there could be alive allocations already, and they could get freed / reused during this test.

Reply:
Oh makes sense, will make it set through `MALLOC_CONF`. To confirm , if `config_limit_usize_gap` is not set, setting `opt_limit_usize_gap` in `MALLOC_CONF` would result in `invalid conf pair`, that doesn't matter right?

Reply:
It still matters since here are tests (e.g. hhvm) that check for output like warnings. You can guard the option with config check this way:
https://github.com/jemalloc/jemalloc/blob/ac279d7e717e6b5f836657fbc525d0975f80a7d0/test/unit/prof_active.sh#L3